precompiles/ArbGasInfo.go

```diff
@@ -162,6 +162,16 @@ func (con ArbGasInfo) GetL1BaseFeeEstimateInertia(c ctx, evm mech) (uint64, erro
 	return c.State.L1PricingState().Inertia()
 }
 
+// GetL1RewardRate gets the L1 pricer reward rate
+func (con ArbGasInfo) GetL1RewardRate(c ctx, evm mech) (uint64, error) {
+	return c.State.L1PricingState().GetRewardsRate()
+}
+
+// GetL1RewardRecipient gets the L1 pricer reward recipient
+func (con ArbGasInfo) GetL1RewardRecipient(c ctx, evm mech) (common.Address, error) {
```

Comment:
One thing I forgot to mention, we need to make sure these methods are only accessible on ArbOS 11 to make sure nodes agree on if these methods exist at any point in time. Search for `GetL1FeesAvailable` in `precompiles/precompile.go` for an example of how to do this.